### DISCUSSION TITLE
Présentations finales

### DISCUSSION THREAD

**Discussion by sjinko** [CUPUM/ecometropole_laurentienne#74] [General]
A discuter

**Comment by sjinko** [comment]
@ArianeViens, voici la conférence de Zhao et Zhao organisé l’année dernière: 

[<img width="953" alt="Screen Shot 2022-12-05 at 18 01 09" src="https://user-images.githubusercontent.com/8132989/205761567-384a243a-9bc9-47ec-b1f5-b2acd5123b09.png">](https://www.youtube.com/watch?v=-uvdG0P0hrs&list=PLzPCds4Ck05jwNMsgbimxrHNmxJ2TX1L4&index=2)

**Reply by anasaboayham** [reply]
> @ArianeViens, voici la conférence de Zhao et Zhao organisé l’année dernière:
> 
> [<img alt="Screen Shot 2022-12-05 at 18 01 09" width="953" src="https://user-images.githubusercontent.com/8132989/205761567-384a243a-9bc9-47ec-b1f5-b2acd5123b09.png">](https://www.youtube.com/watch?v=-uvdG0P0hrs&list=PLzPCds4Ck05jwNMsgbimxrHNmxJ2TX1L4&index=2)

**Comment by sjinko** [comment]
Salut @CUPUM/ame6415_2022, voici l’ordre de présentation et les jury associés:

## Equipe 1
### Jury:
* Heather Braiden (Professeure à l’université de Montréal)
* Pascal Harvey (Directeur du design urbain à Sid Lee Architecture)
* Ange Sauvage (Architecte Atelier TAG)
### Etudiants:
* Marie-Antoinette
* Alexandra
* Arianne

## Equipe 2
### Jury:
* Isabelle Thomas (Professeure à l’université de Montréal)
* Fanny Duguay-Lefebvre (Architecte associée à Civiliti)
* Catherine Beetz (Designer urbain chez Relief Design)
### Etudiants:
* Maryanne
* Flavia
* Ahmed

## Equipe 3
### Jury:
* Christine Robitaille (Architecte associée chez Tiffault Robitaille)
* Guillaume Ethier (Professeur à l’UQAM)
* Geneviève Depelteau (Architecte de paysage à l’Enclume)
### Etudiants:
* Sarah 
* Chloé
* Katherine

**Reply by anasaboayham** [reply]
> Salut @CUPUM/ame6415_2022, voici l’ordre de présentation et les jury associés:
> 
> ## Equipe 1
> ### Jury:
> * Heather Braiden (Professeure à l’université de Montréal)
> * Pascal Harvey (Directeur du design urbain à Sid Lee Architecture)
> * Ange Sauvage (Architecte Atelier TAG)
> 
> ### Etudiants:
> * Marie-Antoinette
> * Alexandra
> * Arianne
> 
> ## Equipe 2
> ### Jury:
> * Isabelle Thomas (Professeure à l’université de Montréal)
> * Fanny Duguay-Lefebvre (Architecte associée à Civiliti)
> * Catherine Beetz (Designer urbain chez Relief Design)
> 
> ### Etudiants:
> * Maryanne
> * Flavia
> * Ahmed
> 
> ## Equipe 3
> ### Jury:
> * Christine Robitaille (Architecte associée chez Tiffault Robitaille)
> * Guillaume Ethier (Professeur à l’UQAM)
> * Geneviève Depelteau (Architecte de paysage à l’Enclume)
> 
> ### Etudiants:
> * Sarah
> * Chloé
> * Katherine

**Comment by MaryanneMonette** [comment]
Salut ! 
Comment déposer un assemblage de moins de 25 mo ? (taille minimale pour les dépôts). Même si je fais un dossier .zip je suis largement au dessus. 

Merci

**Comment by sjinko** [comment]
Quel fichier fait plus de 10 Mo?
________________________________
